core-processor/src/ext.rs

```diff
@@ -615,8 +615,7 @@ impl Externalities for Ext {
             .message_context
             .current()
             .details()
-            .and_then(|d| d.to_reply_details())
-            .map(|d| d.into_reply_to())
+            .and_then(|d| d.to_reply_details().map(Into::into))
```

Comment:
I think it was better previously, as it was more explicit and you could easily find the implementation, which, oppositely, not easy with current `From` impls.